newm-server/build.gradle.kts

```diff
@@ -128,6 +128,10 @@ dependencies {
     "integTestImplementation"(Dependencies.JUnit.JUPITER)
     "integTestRuntimeOnly"("org.junit.platform:junit-platform-launcher")
     "integTestImplementation"(Dependencies.Typesafe.CONFIG)
+
+    runtimeOnly("com.pinterest.ktlint:ktlint-core:0.49.1")
```

Comment:
I don't think these lines need to be here. Can they be removed?

Reply:
These lines are used to run `ktlint --format`, I was getting a lot of warnings about formatting, thoughts?

Reply:
Try running 

`./gradlew ktlF`

Also, the mockito dependency can definitely go.

Reply:
sure thing, what is `./gradlew ktlF` suppose to do? I ran `--help` but didn't see `ktlF` in the list

Reply:
In gradle, you can shorten some commands.

```
./gradlew ktlintFormat
./gradlew kF
```
all do the same thing